File: tourmatefe/types/platform-feedback.ts
import { Account } from "./account"
import { Payment } from "./payment"

export type PlatformFeedback = {
    feedbackId: number,
    accountId: number,
    paymentId: number,
    rating: number,
    content?: string,
    createdAt: string,
    account?: Account
    payment?: Payment
}

export interface CreatePlatformFeedback {
  accountId: number
  paymentId: number
  rating: number
  content?: string
}

export type PlatformFeedbackDto = {
    feedbackId: number,
    accountId: number,
    accountName: string,
    rating: number,
    content?: string,
    createdAt: string,
    paymentId: number
}

File: TourMateBE/Repositories/Models/PlatformFeedback.cs
using System;
using System.Collections.Generic;

namespace Repositories.Models;

public partial class PlatformFeedback
{
    public int FeedbackId { get; set; }

    public int AccountId { get; set; }

    public int PaymentId { get; set; }

    public int Rating { get; set; }

    public string? Content { get; set; }

    public DateTime? CreatedAt { get; set; }

    public virtual Account Account { get; set; } = null!;

    public virtual Payment Payment { get; set; } = null!;
}
